app/src/main/java/org/sopt/and/component/TodayTop20LazyRow.kt

```diff
@@ -0,0 +1,76 @@
+package org.sopt.and.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import org.sopt.and.data.ContentItem
+import org.sopt.and.data.ContentItemViewModel
+
+@Composable
+fun TodayTop20LazyRow() {
+    val contentItemViewModel: ContentItemViewModel = viewModel()
+    val items = contentItemViewModel.items
+
+    LazyRow(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 6.dp),
+        horizontalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        items(items) { item ->
+            val index = items.indexOf(item) + 1
+            TopImageCard(item, index)
+        }
```

Comment:
아래와 같이 바꿔도 좋을 것 같아요~!!
```suggestion
        itemsIndexed(items) { index, item ->
            TopImageCard(item, index)
        }
```

Reply:
오 넵! 이러면 좀더 간결하게 할 수 있을 거 같네요 감사합니다!